source/arch/intel_usg/boards/rdkb_atom/config/comcast/WebpaAgent.xml

```diff
@@ -27,6 +27,21 @@
                 </parameter>
             </parameters>
         </object>
+	<object>
+            <name>X_RDK_Webpa</name>
+            <objectType>object</objectType>
+            <functions>
+                <func_SetParamStringValue>Webpa_SetParamStringValue</func_SetParamStringValue>
```

Comment:
I feel better to have a different function for this as its different object tree.